pkgs/applications/editors/kakoune/plugins/overrides.nix

```diff
@@ -158,4 +158,21 @@ declare-option -hidden str ansi_filter %{'"$out"'/bin/kak-ansi-filter}
       platforms = platforms.all;
     };
   };
+
+  kakoune-catppuccin = buildKakounePluginFrom2Nix {
+    pname = "kakoune-catppuccin";
+    version = "2024-03-29";
+    src = fetchFromGitHub {
+      owner = "catppuccin";
+      repo = "kakoune";
+      rev = "7f187d9da2867a7fda568b2135d29b9c00cfbb94";
+      sha256 = "0vfpkmm111srkxshvchqfayv26qbfs85gifv5460nckww914xh39";
+    };
+    meta = with lib; {
```

Comment:
Would you like to add yourself as maintainer?

```
maintainers = with maintainers; [ philipwilk ];
```

Reply:
Didn't think it necessary bc it's really simple but sure.

Reply:
Yeah, and I forgot to mention the step of adding yourself to the maintainers file. https://nixos.org/manual/nixpkgs/unstable/#var-meta-maintainers

Since [failing to do this breaks eval](https://gist.github.com/GrahamcOfBorg/25f61596f7ee9f7350900d16b08173e1) let's just skip unless you want to maintain more packages in nixpkgs.